packages/svelte/src/internal/client/dev/console-log.js

```diff
@@ -1,30 +1,33 @@
 import { STATE_SYMBOL } from '../constants.js';
 import { snapshot } from '../../shared/clone.js';
 import * as w from '../warnings.js';
+import { untrack } from '../runtime.js';
 
 /**
  * @param {string} method
  * @param  {...any} objects
  */
 export function log_if_contains_state(method, ...objects) {
-	let has_state = false;
-	const transformed = [];
+	return untrack(() => {
+		let has_state = false;
+		const transformed = [];
 
-	for (const obj of objects) {
-		if (obj && typeof obj === 'object' && STATE_SYMBOL in obj) {
-			transformed.push(snapshot(obj, true));
-			has_state = true;
-		} else {
-			transformed.push(obj);
+		for (const obj of objects) {
+			if (obj && typeof obj === 'object' && STATE_SYMBOL in obj) {
```

Comment:
This is probably even less of an issue than #13216 but we had to try catch it because we do `STATE_SYMBOL in obj` it could throw in secure contexts if `obj` is `window`. Should we do it here too?

Reply:
good catch, added one